.release-please-manifest.json

```diff
@@ -1,6 +1,5 @@
 {
   "libs/client-sdk": "3.2.1",
```

Comment:
Theory: release please will now:
(1) Not think that a release has been performed for `0.1.0` (which it hasn't) due to this change
(2) Will assign the next release number as `0.1.0` due to change in `release-please-config.json:23`